data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -0,0 +1,227 @@
+    - eu_nl
+    - eu_at
+    - eu_pl
+    - eu_pt
+    - eu_ro
+    - eu_si
+    - eu_sk
+    - eu_fi
+    - eu_se
+  consent_mechanism: notice_only
+  data_uses:
+    - provide
+  enforcement_level: not_applicable
+  disabled: False
+  has_gpc_flag: False
+  displayed_in_privacy_center: False
+  displayed_in_overlay: True
+  displayed_in_api: False
+- name: Analytics
+  description: This website uses cookies and services to collect information about how you use the website, like which pages you visited and which links you clicked on. None of this information can be used to identify you. It is all aggregated and, therefore, anonymized. Their sole purpose is to improve website functions. This includes cookies from third-party analytics services but are for the exclusive use of the owner of this website.
+  internal_description: These cookies and services collect information about how users use a website, like which pages they visit and which links they click on. None of this information can be used to identify visitors. It is all aggregated and, therefore, anonymized. Their sole purpose is to improve website functions. This includes cookies from third-party analytics services as long as the cookies and services are for your exclusive use.
+  regions:
+    - eu_be
+    - eu_bg
+    - eu_cz
+    - eu_dk
+    - eu_de
+    - eu_ee
+    - eu_ie
+    - eu_el
+    - eu_es
+    - eu_fr
+    - eu_hr
+    - eu_it
+    - eu_cy
+    - eu_lv
+    - eu_lt
+    - eu_lu
+    - eu_hu
+    - eu_mt
+    - eu_nl
+    - eu_at
+    - eu_pl
+    - eu_pt
+    - eu_ro
+    - eu_si
+    - eu_sk
+    - eu_fi
+    - eu_se
+  consent_mechanism: opt_in
+  data_uses:
+    - improve.system
+  enforcement_level: frontend
+  disabled: False
+  has_gpc_flag: False
+  displayed_in_privacy_center: False
+  displayed_in_overlay: True
+  displayed_in_api: False
+- name: Marketing
+  description: This website uses cookies and services to track your online activity to help advertisers deliver more relevant advertising or to limit how many times you see an ad. These services can share that information with other organizations or advertisers. These are persistent cookies and almost always of from third-parties.
+  internal_description: These cookies and services track online activity of your users to help advertisers deliver more relevant advertising or to limit how many times users see an ad. These services can share that information with other organizations or advertisers. These are persistent cookies and almost always of from third-parties
+  regions:
+    - eu_be
+    - eu_bg
+    - eu_cz
+    - eu_dk
+    - eu_de
+    - eu_ee
+    - eu_ie
+    - eu_el
+    - eu_es
+    - eu_fr
+    - eu_hr
+    - eu_it
+    - eu_cy
+    - eu_lv
+    - eu_lt
+    - eu_lu
+    - eu_hu
+    - eu_mt
+    - eu_nl
+    - eu_at
+    - eu_pl
+    - eu_pt
+    - eu_ro
+    - eu_si
+    - eu_sk
+    - eu_fi
+    - eu_se
+  consent_mechanism: opt_in
+  data_uses:
+    - advertising.third_party.personalized
```

Comment:
Data uses are a required item for creating a privacy notice so I've had to omit all privacy notices without them.

Reply:
@mfbrown are you sure we shouldn't wait to release this until Fideslang 1.4?